src/mongoose_cowboy_metrics_mw_after.erl

```diff
@@ -0,0 +1,121 @@
+%%==============================================================================
+%% Copyright 2016 Erlang Solutions Ltd.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%% http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%
+%% @doc Cowboy middleware updating metrics related to request handling
+%%
+%% This middleware needs to run after `mongoose_cowboy_metrics_mw_before' and  `cowboy_router'
+%% middleware. However, it does not need to run after `cowboy_hander' middleware because metrics are
+%% updated in `onresponse' callback which is called whenever the request is sent.
```

Comment:
Should there be `whenever the response is sent` instead of `request`?

Reply:
Yes, thanks!